libs/authentication/website/src/lib/login-form/login-form-container.tsx

```diff
@@ -25,12 +29,24 @@ export function LoginFormContainer({className}: LoginFormContainerProps) {
         loginWithCredentials({
           variables: {email, password}
         })
+
+        if (afterLoginCallback) {
+          afterLoginCallback()
+        }
       }}
       loginWithCredentials={withCredentials}
       onSubmitLoginWithEmail={email => {
         loginWithEmail({
           variables: {email}
         })
+
+        if (passRedirectCookie) {
```

Comment:
passRedirectCookie & afterLoginCallback do the same thing. Delete passRedirectCookie